docs/data-operations-manual/How-To-Guides/Maintaining/Assign-entities.md

```diff
@@ -35,4 +28,21 @@ digital-land assign-entities collection/tree-preservation-order/resource/4e0de67
 ```
 
 3. **Check results**  
-   Confirm the desired result in the lookup.csv file, the amount of entities that needed to be assigned should be the same amount that have been added in the lookup file.
\ No newline at end of file
+   Confirm the desired result in the lookup.csv file, the amount of entities that needed to be assigned should be the same amount that have been added in the lookup file.
+
+## Batch assigning entities
+
+1. **Download issue summary table**
+   Get the issue summay table from the config-manager for ODP data [here](https://config-manager-prototype.herokuapp.com/reporting/odp-summary/issue) by clicking the `Download Current Table`
```

Comment:
A small bit of style feedback: I think links out to other content are generally a bit easier to understand and follow when the hyperlink surrounds the words describing the link, rather than using something like "here". So this could just be:

Get the issue summary table from the [config-manager for ODP data](https://config-manager-prototype.herokuapp.com/reporting/odp-summary/issue) by clicking the `Download Current Table`.

See [these principles from writethedocs](https://www.writethedocs.org/guide/writing/docs-principles/#skimmable) for an explainer and some other useful advice.

Reply:
Good spot, fixed!